.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+# dependabot.yaml reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+#
+# Notes:
+# - Status and logs from dependabot are provided at
+#   https://github.com/jupyterhub/jupyterhub/network/updates.
```

Comment:
I just noticed that we might need to make an edit to the link provided for dependabot's status and logs.

Reply:
Ah nice catch!!